### PR TITLE
Fix the text rendering Bug

### PR DESCRIPTION
**Pull request by samcake**
I broke the text rendering in my previous PR, This is fixing it
Simply make sure the alpha is 1 before going into the packDeferredFragmentLightmap function
 Also fixed the color which was not gamma corrected

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 193b98f24fb35580faed499bdcb952712d96ba1a available at: https://gist.github.com/7f642d7603ab11051812<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7256.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7256.dmg

**Comment by zzmp** [comment]
Does this make text always opaque? If so, isn't that a feature degradation?

File: libraries/render-utils/src/sdf_text3D.slf [hunk 1]
```diff
@@ -49,19 +46,15 @@ void main() {
...
     
-    // final color
-   /* _fragColor0 = vec4(Color.rgb, Color.a * a);
-    _fragColor1 = vec4(normalize(_normal.xyz), 0.0) * 0.5 + vec4(0.5, 0.5, 0.5, 0.5);
-    _fragColor2 = vec4(Color.rgb, 10 / 128.0);
-    */
     packDeferredFragmentLightmap(
          normalize(_normal),
-         Color.a * a,
-         Color.rgb,
```
Comment (jherico) [inline libraries/render-utils/src/sdf_text3D.slf]: Won't this break support for colored text?

Comment (samcake) [inline libraries/render-utils/src/sdf_text3D.slf]: NO because the color goes in the emissive still

**Comment by samcake** [comment]
@zzmp No change with that, The fragment is discarded the same way as before when the alpha reaches <= 0.1
It will probably be different in the case this shader is used during the non deferred passes which is another story...
Just going for the quick fix here

**Comment by bluetoe** [comment]
I tried this with text entity on pr domain and on current merged alpha domain with the PR.. looks ok. all text readable now where it was not before, although where I had noticed it most before was when changing graphics setting, which has been changed since then.. but all looks ok now, at least to me, cept the text now is kind of big and thick.. sort of, but yes much better now.

**Pull request merged**